resources/assets/uad_lists.json

```
@@ -36562,5 +36561,374 @@
     "neededBy": [],
     "labels": [],
     "removal": "Recommended"
+  },
+  {
+    "id": "com.huawei.airlink",
+    "list": "Oem",
+    "description": "Air link service, its problably needed for Wireless Projection.\nNo activities, only more debugging stuff, location things, logs",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.huawei.android.airsharing",
+    "list": "Oem",
+    "description": "Wireless Projection\nMiracast, Require Huawei Mobile Services, also its so bloated, also found unused frameworks HUAWEI GameCenter",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.huawei.audioaccessorymanager",
+    "list": "Oem",
+    "description": "Audio Accessory Manager\nA lot stuff: huawei health, huawei music, earphones.\nProblably useful when you got HUAWEI FreeBuds.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
+  },
+  {
+    "id": "com.huawei.calculator",
+    "list": "Oem",
+    "description": "Calculator app",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Advanced"
+  },
+  {
+    "id": "com.huawei.coauthservice",
+    "list": "Oem",
+    "description": "HwCoAuthService\nFounded Applock, lockscreen password, face recognition things, fingerprint, not sure if its worth removing it",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Expert"
+  },
+  {
+    "id": "com.huawei.contentsensor",
+    "list": "Oem",
+    "description": "ContentSensor\nIts app only for call logs",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Advanced"
```
Call logs can be seen as something essential.